src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -345,6 +345,13 @@ export const QueryResultPane = () => {
 
     const renderGridPanel = () => {
         const grids = [];
+        // execution plans only load after reading the resulting xml showplan
+        // of the query. therefore, it updates the state once the results
+        // are loaded, which causes a rendering loop if the grid
+        // gets refreshed
+        if (!metadata?.isExecutionPlan) {
```

Comment:
@laurennat Do we still need this if check here?

Reply:
nope we don't! take out the check and it should be fine.